README.md

```diff
@@ -103,90 +106,65 @@ These instructions assume you've used the
 used a [custom installation](#advanced-client-installation) for the client
 and/or blockchain software, adapt as necessary.
 
-All commands listed below are of the Linux variety, and assume you already
-`cd` into the `~/decred` directory created by **dcrinstall**.
-
 ### Prerequisites
 
-1. dcrd, dcrwallet, and bitcoind should be running and synced.
-
-2. It is highly recommended that you create separate accounts for trading.
-
-#### Creating a trading account for Decred
-
-For Decred, start dcrwallet and create an account in your terminal or console
-using the **dcrctl** utility.
-
-`./dcrctl --wallet createnewaccount dex`
-
-Your dex trading account uses the wallet password you've set up with
-**dcrinstall** or `./dcrwallet --create`.
-
-Get an address for the new account, and transfer some funds. You'll need to
-deposit at least enough to cover the registration fee and on-chain fees for the
-registration transaction. The registration process will inform you how much to
-pay. At the time of writing, it was 1 DCR. Of course, if you plan to sell any
-DCR, you should deposit that too.
-
-`./dcrctl --wallet getnewaddress dex`
+External wallet software is not required for most assets, just the DEX client!
+The native light wallets are simplest and best option for most users. But if
+using external wallets, they should be running and synced before staring DEX.
```

Comment:
The native light wallets are simplest and **the** best option for most users.

I think. I may be forgetting English.